src/content/docs/telemetry-data-platform/ingest-apis/report-metrics-metric-api.mdx

```diff
@@ -24,14 +24,14 @@ Metric data is submitted to New Relic through an HTTP POST request. Each request
 
 Follow this example to send your first metric data points to New Relic:
 
-1. Find or register an [Insert API key](/docs/insights/insights-data-sources/custom-data/send-custom-events-event-api#register) for your account: Go to: **[one.newrelic.com](https://one.newrelic.com) > [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary#account-dropdown) > Account settings > API keys**, and select **Insights API keys**.
-2. Insert your API key into the following JSON, and then send the JSON to our [endpoint](#api-endpoint).
+1. Get the [license key](/docs/apis/intro-apis/new-relic-api-keys/#ingest-license-key) for the account you want to report data to.
```

Comment:
Or "data-reporting account"?

Reply:
just a heads up: I'm not making that change because it's not technically a "data-reporting account"; it's more accurate to say that data-reporting apps or hosts are reporting data to the account, while 'data-reporting account' makes it sound like the account is reporting data somewhere. I get what you mean that it could be more succinct but I was having trouble finding a way that was conversationally clear and didn't confuse me more.